### PR TITLE
update UI to reflect final mockups

### PR DESCRIPTION
**Pull request by fuskovic**
What: Updates the mark-up and styling of the UI to reflect designer mocks. Also fixes serving of static assets from the mux.

Why: So that the referral admin portal reflects the theme marketing admins expect to see when using the service.

Thanks for submitting a PR!

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Could the PR be broken into smaller PRs?
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?

### CODE REVIEW

File: satellite/marketing/marketingweb/server.go [hunk 1]
```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/zeebo/errs"
 	"go.uber.org/zap"
 	"golang.org/x/sync/errgroup"
+	"github.com/gorilla/mux"
```
Comment (egonelbre) [inline satellite/marketing/marketingweb/server.go]: Why do you need gorilla/mux here?

Comment (fuskovic) [reply satellite/marketing/marketingweb/server.go]: @egonelbre  So I can use PathPrefix on line 63. That fixes the serving of static assets. Also, I have future code where I need to pass params to the endpoints which gorilla/mux handles nicely.

Comment (egonelbre) [reply satellite/marketing/marketingweb/server.go]: The previous one http.StripPrefix and http.Handle was working.

Comment (fuskovic) [reply satellite/marketing/marketingweb/server.go]: @egonelbre  The previous one actually wasn't working. I told @VinozzZ I would fix it in my PR since the scope of her PR was only to open the port. Also I have code I haven't put up yet where I use mux to pass params to endpoints like `/updateOffer/r{offerID}` which I like. Is that ok ? If not what do you suggest ?

File: web/marketing/css/style.css [hunk 2]
```diff
@@ -6,26 +6,117 @@ html{
...
 
+.btn{
+    height:48px;
+}
+
+hr{
+    margin-top:-5px;
+    margin-left: -15px;
+    margin-bottom: 0px;
+    width: 1102px !important;
```
Comment (VinozzZ) [inline web/marketing/css/style.css]: Why do we need all the `!important` here?

File: web/marketing/css/style.css [hunk 3]
```diff
@@ -6,26 +6,117 @@ html{
...
+.col-heading {
+    font-family: 'Roboto';
+    font-style: normal;
+}
+
+.data-row{
+    font-weight: normal;
+    font-size: 14px;
+    line-height: 49px;
+    letter-spacing: -0.100741px;
```
Comment (VinozzZ) [inline web/marketing/css/style.css]: Is this number coming from the mockup?

File: web/marketing/css/style.css [hunk 4]
```diff
@@ -6,26 +6,117 @@ html{
...
+
+.ref-admin,h3{
+    font-family: 'Poppins';
+    font-style: normal;
+}
+
+.ref-admin{
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 130.02%;
```
Comment (VinozzZ) [inline web/marketing/css/style.css]: `line-height` shouldn't be a percentage

File: web/marketing/css/style.css [hunk 5]
```diff
@@ -38,4 +129,49 @@ body{
 
 .toggler:hover{
     cursor:pointer;
+}
+
+#foModalLabel, 
+#roModalLabel{
+    font-family: 'Poppins' !important;
```
Comment (VinozzZ) [inline web/marketing/css/style.css]: why `!important` if it's using id selector?

File: web/marketing/pages/freeOffers.html [hunk 6]
```diff
@@ -1,51 +1,58 @@
...
+            <div class="col col-heading">Expiration</div>
+            <div class="col col-heading">Status</div>
+            <div class="col col-heading edit-offer"></div>
+        </div>
+        <hr>
+        <div class="row offer-heading ">
+            <p class="offer-type">DEFAULT&nbsp;OFFER</p>
+        </div>
+        <div class="row data-row">
+            <div class="col ml-3">May Referral</div>
```
Comment (VinozzZ) [inline web/marketing/pages/freeOffers.html]: Is the dynamic template going to be built in another PR? Right now all the data is hard-coded

File: web/marketing/css/style.css (hunk 3 above)
Comment (fuskovic) [reply web/marketing/css/style.css]: It is.

File: web/marketing/pages/freeOffers.html (hunk 6 above)
Comment (fuskovic) [reply web/marketing/pages/freeOffers.html]: Yes I already have all that done. It will be included in the Create-offer PR. I just wanted to keep the code separate to have smaller PR's.

File: web/marketing/css/style.css (hunk 5 above)
Comment (fuskovic) [reply web/marketing/css/style.css]: I forgot to remove it after I got the static asset handler working. Just pushed the changes.

File: web/marketing/css/style.css (hunk 4 above)
Comment (fuskovic) [reply web/marketing/css/style.css]: done

File: web/marketing/css/style.css (hunk 2 above)
Comment (fuskovic) [reply web/marketing/css/style.css]: No. I removed it.

File: web/marketing/css/style.css [hunk 7]
```diff
@@ -38,4 +129,49 @@ body{
 
 .toggler:hover{
     cursor:pointer;
+}
+
+#foModalLabel, 
+#roModalLabel{
```
Comment (egonelbre) [inline web/marketing/css/style.css]: What's `fo` and `ro`? Any reason these names have different conventions than rest of the names?

File: satellite/marketing/marketingweb/server.go (hunk 1 above)
Comment (egonelbre) [reply satellite/marketing/marketingweb/server.go]: ok, sounds good

File: web/marketing/css/style.css (hunk 7 above)
Comment (fuskovic) [reply web/marketing/css/style.css]: fo is free offer and ro is referral offer. I tried to keep the names short.

Comment (egonelbre) [reply web/marketing/css/style.css]: I would go with `#free-offer-modal-label` or `#free-offer-label` or something like that. Previously when looking at the filenames I got confused about `fo` and `ro` as well.

Comment (egonelbre) [reply web/marketing/css/style.css]: Alternatively shorten the part that people are more used to e.g. `lbl`, but I really don't see a good reason to shorten this.

Comment (fuskovic) [reply web/marketing/css/style.css]: OK, I'll update the filenames and id's.

File: satellite/marketing/marketingweb/server.go [hunk 8]
```diff
@@ -96,8 +98,8 @@ func (s *Server) parseTemplates() (err error) {
 		filepath.Join(s.templateDir, "home.html"),
 		filepath.Join(s.templateDir, "refOffers.html"),
```
Comment (egonelbre) [inline satellite/marketing/marketingweb/server.go]: referral-offers.html

File: satellite/marketing/marketingweb/server.go [hunk 9]
```diff
@@ -96,8 +98,8 @@ func (s *Server) parseTemplates() (err error) {
...
 		filepath.Join(s.templateDir, "freeOffers.html"),
```
Comment (egonelbre) [inline satellite/marketing/marketingweb/server.go]: `free-offers.html`

File: satellite/marketing/marketingweb/server.go [hunk 10]
```diff
@@ -96,8 +98,8 @@ func (s *Server) parseTemplates() (err error) {
...
-		filepath.Join(s.templateDir, "roModal.html"),
-		filepath.Join(s.templateDir, "foModal.html"),
+		filepath.Join(s.templateDir, "ref-offers-modal.html"),
```
Comment (egonelbre) [inline satellite/marketing/marketingweb/server.go]: `referral-offers-modal.html`

File: web/marketing/css/style.css [hunk 11]
```diff
@@ -6,26 +6,117 @@ html{
...
+.col-heading{
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 49px;
+    align-items: center;
+    letter-spacing: -0.100741px;
+    color: #96989E;
+}
+
+.ref-admin,h3{
```
Comment (egonelbre) [inline web/marketing/css/style.css]: `referral-admin`

File: web/marketing/css/style.css [hunk 12]
```diff
@@ -6,26 +6,117 @@ html{
...
+    font-family: 'Poppins';
+    font-style: normal;
+}
+
+.ref-admin{
```
Comment (egonelbre) [inline web/marketing/css/style.css]: `referral-admin`

Comment (fuskovic) [reply web/marketing/css/style.css]: done

File: web/marketing/css/style.css (hunk 11 above)
Comment (fuskovic) [reply web/marketing/css/style.css]: done

File: satellite/marketing/marketingweb/server.go (hunk 10 above)
Comment (fuskovic) [reply satellite/marketing/marketingweb/server.go]: done

File: satellite/marketing/marketingweb/server.go (hunk 9 above)
Comment (fuskovic) [reply satellite/marketing/marketingweb/server.go]: done

File: satellite/marketing/marketingweb/server.go (hunk 8 above)
Comment (fuskovic) [reply satellite/marketing/marketingweb/server.go]: done

File: web/marketing/pages/home.html [hunk 13]
```diff
@@ -1,16 +1,19 @@
...
 {{define "main"}}
 <div class="container">
-    <div class="row mt-4 mb-4">
-        <h3 class="m-4 text-dark">Referral Credit Offers</h3>
-        <button type="button" class="btn btn-outline-dark m-4" data-toggle="modal" data-target="#roModal">+ Create Referral Credit</button>
+    <div class="row mt-4 mb-4 d-flex justify-content-between">
+        <h3 class="mt-4">Referral Credit Offers</h3>
+        <button type="button" class="btn btn-primary mt-4" data-toggle="modal" data-target="#ref-offers-modal">+ Create Referral Credit</button>
     </div>
     {{template "refOffers" .}}
```
Comment (egonelbre) [inline web/marketing/pages/home.html]: still using shorthands

File: web/marketing/pages/home.html [hunk 14]
```diff
@@ -1,16 +1,19 @@
...
-    {{template "roModal" .}}
-    <div class="row mt-4 mb-4">
-        <h3 class="m-4 text-dark">Free Credit Offers</h3>
-        <button type="button" class="btn btn-outline-dark m-4" data-toggle="modal" data-target="#foModal">+ Create Free Credit</button>
+    {{template "refOffersModal" .}}
```
Comment (egonelbre) [inline web/marketing/pages/home.html]: still using shorthands

File: web/marketing/pages/home.html [hunk 15]
```diff
@@ -1,16 +1,19 @@
...
+See LICENSE for copying information. -->
+
 {{define "main"}}
 <div class="container">
-    <div class="row mt-4 mb-4">
-        <h3 class="m-4 text-dark">Referral Credit Offers</h3>
-        <button type="button" class="btn btn-outline-dark m-4" data-toggle="modal" data-target="#roModal">+ Create Referral Credit</button>
+    <div class="row mt-4 mb-4 d-flex justify-content-between">
+        <h3 class="mt-4">Referral Credit Offers</h3>
+        <button type="button" class="btn btn-primary mt-4" data-toggle="modal" data-target="#ref-offers-modal">+ Create Referral Credit</button>
```
Comment (egonelbre) [inline web/marketing/pages/home.html]: still using shorthands

File: web/marketing/pages/home.html (hunk 13 above)
Comment (fuskovic) [reply web/marketing/pages/home.html]: Fixed before merging.

File: web/marketing/pages/home.html (hunk 14 above)
Comment (fuskovic) [reply web/marketing/pages/home.html]: Fixed before merging.

File: web/marketing/pages/home.html (hunk 15 above)
Comment (fuskovic) [reply web/marketing/pages/home.html]: Fixed before merging.
